### PR TITLE
Change order of edit button

### PR DESCRIPTION
**Pull request by mdruzhin**
### Feature

[PCOM-5933](https://jira.ae.com/browse/PCOM-5933)

### Summary

Changed order of edit button on CustomerDetails screen

### CODE REVIEW

**Comment by chenson42** [comment]
@gwilmer @mmichalek Does the order of matter to other customers?

**Review by chenson42 — APPROVED** [review]

**Review by jchirico8 — CHANGES_REQUESTED** [review]
These buttons are too hardcoded. I think it started off innocently with a done and unlink customer button but it's gotten out of control. AE doesn't have to do this now, but we need to refactor these to come from a list of actions instead of hardcoding each type of button

**Comment by jchirico8** [comment]
Since AE is pushing such a tight timeline I would tell them to deal with it's existing location and they can use a CR to properly fix it later

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html [hunk 1]
```diff
@@ -147,40 +147,12 @@ <h3 class="title">{{ item.title }}</h3>
...
-    <app-secondary-button responsive-class
-                          class="unlink"
-                          *ngIf="screen.unlinkButton"
-                          [disabled]="!screen.unlinkButton?.enabled"
-                          [actionItem]="screen.unlinkButton"
-                          (actionClick)="doAction(screen.unlinkButton)"
-                          (click)="doAction(screen.unlinkButton)">
-        <span>{{screen.unlinkButton.title}}</span>
-    </app-secondary-button>
-    <app-primary-button responsive-class
```
Comment (jchirico8) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html:177]: You need to keep this one primary button that's on the right

**Review by blubaugha — APPROVED** [review]

**Review by jchirico8 — APPROVED** [review]

**Pull request merged**